lib/models/builder.js

```diff
@@ -26,7 +27,11 @@ class Builder extends CoreObject {
   constructor(options) {
     super(options);
 
+    this.broccoli2 = Boolean(options.broccoli2 === undefined ? experiments.BROCCOLI_2 : options.broccoli2);
+    this.systemTemp = Boolean(options.systemTemp === undefined ? experiments.SYSTEM_TEMP : options.systemTemp);
```

Comment:
Same RE: experiment checks. In general it should not be possible to force these experiments to true without `process.env.EMBER_CLI_*` is set...